cmd/ctr/restore.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	gocontext "context"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/containerd/containerd/api/services/execution"
+	"github.com/containerd/containerd/api/types/mount"
+	"github.com/containerd/containerd/images"
+	protobuf "github.com/gogo/protobuf/types"
+	digest "github.com/opencontainers/go-digest"
+	specs "github.com/opencontainers/image-spec/specs-go"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/urfave/cli"
+)
+
+var restoreCommand = cli.Command{
+	Name:  "restore",
+	Usage: "restore a container from an existing checkpoint",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "id",
+			Usage: "id of the container",
+		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Usage: "runtime name (linux, windows, vmware-linux)",
+			Value: "linux",
+		},
+		cli.BoolFlag{
```

Comment:
ya, I will have to add this information from the original container creation to the restore.  I'm waiting on the container metadata store before doing this then these flags will go away.